spec/models/ring_spec.rb

```diff
@@ -4,14 +4,16 @@
   describe "#link" do
     it "adds a new redirection while preserving the ring" do
       new_redirection = build(:redirection)
```

Comment:
I think it's generally better to use build where you can because the you don't have to hit the DB as much.